### PR TITLE
Make sure UIAlertViews are always rendered vertically

### LINKED ISSUE #2611: Islandwood House Demo Completed popup is rotated 90 degrees.

### PR DESCRIPTION
**Pull request by jaredhms**
WoCDisplayMode uses a 'presentationTransform' to rotate the presentation surface for certan apps, but we don't want to use that transform to rotate UIAlertViews (else they render sideways if the app is rotated).  This will be handled automatically when we move UIAlertView over Xaml (as we've already done for UIActionSheet), but for now we need to undo the 'presentationTransform' rotation for UIAlertViews.

Fixes #2611.

### CODE REVIEW

File: Frameworks/UIKit/UIAlertView.mm [hunk 1]
```diff
@@ -1,641 +1,671 @@
...
+    alertPriv->_titleLabel = nil;
+    alertPriv->_message = nil;
+    alertPriv->_messageLabel = nil;
+    alertPriv->_cancelText = nil;
+    alertPriv->_cancelView = nil;
+
+    delete alertPriv;
+
+    [super dealloc];
+}
```
Comment (DHowett-MSFT) [inline Frameworks/UIKit/UIAlertView.mm]: This file is impossible to review; did the line endings change?

Comment (yiyang-msft) [inline Frameworks/UIKit/UIAlertView.mm]: ?? I don't have problem to view this

---
In reply to: [113998799](https://github.com/Microsoft/WinObjC/pull/2652#discussion_r113998799) [](ancestors = 113998799)

File: Frameworks/UIKit/UIAlertView.mm [hunk 2]
```diff
@@ -1,641 +1,671 @@
...
+/**
+ @Status Interoperable
+*/
+- (BOOL)isVisible {
+    return alertPriv->_isVisible;
+}
+
+- (void) _handleRotation {
+    // We need to offset the 'presentationTransform' used in WOCDispalyMode, because we always want to render alerts vertically.
+    // TODO: We'll remove this when we switch UIAlertView over to a Xaml ContentDialog, since rotation will be handled for us.
```
Comment (yiyang-msft) [inline Frameworks/UIKit/UIAlertView.mm]: nit: if we have WI#, we can put it here so that we don't forget to remove this.

**Comment by yiyang-msft** [comment]
:shipit:

File: Frameworks/UIKit/UIAlertView.mm (hunk 2 above)
Comment (jaredhms) [inline Frameworks/UIKit/UIAlertView.mm]: Basically this whole file will be gutted at that time.  I don't think this particular backlog item has been filed on github though...

**Comment by tadam-msft** [comment]
:shipit:

File: Frameworks/UIKit/UIAlertView.mm (hunk 1 above)
Comment (jaredhms) [inline Frameworks/UIKit/UIAlertView.mm]: Doh - nice catch.  Reinstall of git seems to have stomped my settings.

---
In reply to: [113998959](https://github.com/Microsoft/WinObjC/pull/2652#discussion_r113998959) [](ancestors = 113998959,113998799)

File: Frameworks/UIKit/UIAlertView.mm [hunk 3]
```diff
@@ -396,6 +397,32 @@ - (BOOL)isVisible {
     return alertPriv->_isVisible;
 }
 
+- (void) _handleRotation {
+    // We need to offset the 'presentationTransform' used in WOCDispalyMode, because we always want to render alerts vertically.
```
Comment (DHowett-MSFT) [inline Frameworks/UIKit/UIAlertView.mm]: nit: "dispaly"

**Pull request merged**